src/Content.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Bolt;
 
+use Maid\Maid;
 use Silex;
 use Symfony\Component\Filesystem\Filesystem;
 use Bolt\Library as Lib;
```

Comment:
Alphabetical order on the use statements… Last time I checked 'B' came before 'M' and 'S'.